xc/common/primitives/ipad/ipad.model.xml

```diff
@@ -0,0 +1,7 @@
+<models>
+  <model name="IPAD_VPR">
```

Comment:
This will require `never_prune="true"` so that VPR does not remove this. From VPR perspective its a leaf cell with inputs only.